packages/nuxi/src/commands/dev.ts

```diff
@@ -44,20 +44,22 @@ export default defineNuxtCommand({
 
     await setupDotenv({ cwd: rootDir, fileName: args.dotenv })
 
+    const { loadNuxt, loadNuxtConfig, buildNuxt } = await loadKit(rootDir)
+
+    const config = await loadNuxtConfig({})
```

Comment:
```suggestion
    const config = await loadNuxtConfig({ rootDir, dev: true, ready: false })
```

Reply:
I did do this originally, however remove it as the following error occurs:

```
Argument of type '{ rootDir: string; dev: boolean; ready: boolean; }' is not assignable to parameter of type 'LoadConfigOptions<NuxtConfig>'.
  Object literal may only specify known properties, and 'rootDir' does not exist in type 'LoadConfigOptions<NuxtConfig>'.ts(2345)
```

`dev` and `ready` also do not exist.

Reply:
quite right - pushed the correct syntax for `loadNuxtConfig`